### PR TITLE
Allow for job dependencies

### PR DESCRIPTION
**Pull request by niklassiemer**
Momentan 'nur' für slurm

### CODE REVIEW

**Comment by niklassiemer** [comment]
ping @jan-janssen

File: pysqa/cmd.py [hunk 1]
```diff
@@ -83,6 +85,11 @@ def command_line(argv):
                 mode_status = True
             elif opt in ("-l", "--list"):
                 mode_list = True
+            elif opt in ("-b", "--dependency"):
```
Comment (jan-janssen) [inline pysqa/cmd.py:88]: I am not sure if `-b` is the best choice for dependencies. But as I do not have a better suggestion either as `-d` is already in use for delete, I am fine with this.

**Review by jan-janssen — APPROVED** [review]
Looks good to me

**Comment by jan-janssen** [comment]
> ping @jan-janssen

I am sorry for the delay, somehow this slipped through.

**Comment by niklassiemer** [comment]
Nevermind, this was not pressing for me :)

**Pull request merged**